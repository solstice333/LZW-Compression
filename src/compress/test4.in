Hello, this is a test of the new LZW compressor.  This test should not result
in a refresh of the dictionary, though we will probably get codes that are
at least 9 bits from it, which is a good validation of the basic LZW function,
For a short program, this thing can cause a lot of sleepless nights
Hello, this is a test of the new LZW compressor.  This test should not result
in a refresh of the dictionary, though we will probably get codes that are
at least 9 bits from it, which is a good validation of the basic LZW function,
For a short program, this thing can cause a lot of sleepless nights
Hello, this is a test of the new LZW compressor.  This test should not result
in a refresh of the dictionary, though we will probably get codes that are
at least 9 bits from it, which is a good validation of the basic LZW function,
For a short program, this thing can cause a lot of sleepless nights
