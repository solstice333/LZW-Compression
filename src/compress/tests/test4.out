Space after LZWCmpStop for test4.in: 75023
Final space: 0
